src/EventListenerService/Handlers/VoiceStateUpdateHandler.cs

```diff
@@ -47,6 +44,11 @@ public async ValueTask HandleAsync(VoiceState voiceState)
             return;
         }
 
+        if (await scopedWojtusContext.UserEmojis.FindAsync(voiceState.UserId) is null) {
+            logger.LogDebug("No emoji configured for user {UserId}", voiceState.UserId);
```

Comment:
I'm thinking if maybe we should have a default emoji like 🤷 or whatever just so everyone is included in the description
if you think that make sense you can add ticket for that